src/Grpc/JsonTranscoding/test/Microsoft.AspNetCore.Grpc.JsonTranscoding.Tests/ConverterTests/JsonConverterReadTests.cs

```diff
@@ -451,6 +502,19 @@ public void MapKeyBool()
         AssertReadJson<HelloRequest>(json);
     }
 
+    [Fact]
+    public void MapKeyBool_CaseInsensitive()
+    {
+        var json = @"{
+  ""mapKeybool"": {
+    ""TRUE"": ""value1"",
+    ""FALSE"": ""value2""
+  }
+}";
+
+        AssertReadJson<HelloRequest>(json, serializeOld: false);
```

Comment:
```suggestion
        AssertReadJson<HelloRequest>(json, serializeOld: false, settings: new GrpcJsonSettings { FieldNamesCaseInsensitive = true });
```

Reply:
Added a comment explaining why that's not required in this test.

Reply:
Ah, I see. Thanks!